style.css

```diff
@@ -0,0 +1,681 @@
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.nav-menu-pop li a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  justify-content: center;
+  padding-left: 3%;
+  padding-right: 5%;
+  padding-bottom: 17px;
+  margin-top: -35px;
+  margin-bottom: -115px;
+}
+
+.main-about {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  justify-content: center;
+  align-items: center;
+  padding-left: 3%;
+  padding-right: 5%;
+  padding-bottom: 17px;
+  padding-top: 25px;
+  gap: 13px;
+}
+
+.main-sub-title {
+  font-family: "Lato", sans-serif;
+  font-size: 23px;
+  font-weight: 400;
+  color: rgb(236, 82, 66);
+  padding-block: 1px;
+  margin: 0;
+}
+
+.main-about-sub-title {
+  font-family: "Lato", sans-serif;
+  font-size: 23px;
+  font-weight: 400;
+  color: rgb(236, 82, 66);
+  padding-block: 1px;
+  margin: 0;
+}
+
+.logo,
+.menu-options {
+  display: none;
+}
+
+.events-about-img {
+  width: 100%;
+}
+
+.main-title {
+  font-family: "Lato", sans-serif;
+  font-size: 29px;
+  font-weight: 900;
+  line-height: 30px;
+  color: rgb(236, 82, 66);
+  margin: 0;
+}
+
+.main-about-title {
+  font-family: "Lato", sans-serif;
+  font-size: 25px;
+  font-weight: 900;
+  line-height: 30px;
+  text-align: center;
+  color: rgb(236, 82, 66);
+  margin: 0;
+}
+
+.main-decription {
+  background-color: rgb(247, 247, 247);
+  border: 3px solid rgb(255, 255, 255);
+  border-radius: 5px;
+  width: 90%;
+  padding: 20px;
+  font-family: "Lato", sans-serif;
+  color: rgb(111, 112, 116);
+  margin-top: 10px;
+  font-size: 14px;
+  line-height: 23px;
+}
+
+.main-about-decription {
+  background-color: rgb(255, 255, 255);
+  border: 3px solid rgb(255, 255, 255);
+  border-radius: 5px;
+  width: 85%;
+  padding: 20px;
+  font-family: "Lato", sans-serif;
+  color: rgb(111, 112, 116);
+  margin-top: 10px;
+  font-size: 14px;
+  text-align: center;
+  line-height: 23px;
+}
+
+.main-about-desc-mid {
+  background-color: rgb(255, 255, 255);
+  border: 3px solid rgb(255, 255, 255);
+  border-radius: 5px;
+  width: 85%;
+  padding-inline: 20px;
+  margin: 0;
+  font-family: "Lato", sans-serif;
+  color: rgb(111, 112, 116);
+  font-size: 14px;
+  text-align: center;
+  line-height: 23px;
+}
+
+.main-about-info {
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  color: rgb(111, 112, 116);
+  text-align: center;
+  margin-top: 15px;
+  line-height: 23px;
+}
+
+.my-logo {
+  width: 70%;
+}
+
+.img-container {
+  display: flex;
+  width: 93%;
+  border: 1px solid rgb(198, 207, 198);
+  padding: 10px;
+  height: 110px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  justify-content: center;
+}
+
+.schedule {
+  font-family: "Lato", sans-serif;
+  font-size: 25px;
+  font-weight: 650;
+  color: #272a31;
+  align-self: flex-start;
+  padding-left: 3%;
+  margin: 0;
+}
```

Comment:
Kindly make sure your project sticks as much as possible to the template provided. This text should have a font-weight higher. Kindly change this. :eyes: :eyes: 
Your design.
![image](https://user-images.githubusercontent.com/98363075/196956892-12db7538-7330-4f36-8eee-03cffdd5fd4e.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/196957004-768619aa-4a4f-43d7-bede-733dd64f4af8.png)